pennylane/tape/circuit_graph.py

```diff
@@ -29,6 +30,11 @@ class TapeCircuitGraph(CircuitGraph):
     def __init__(self, ops, obs, wires):
         self._operations = ops
         self._observables = obs
+
+        for m in self._observables:
+            if m.return_type is qml.operation.State:
+                m._wires = wires  # pylint: disable=protected-access
```

Comment:
When creating the circuit graph, we modify `State` returns to have the same wires as the device, so that it displays correctly.

Reply:
Oh, I missed this comment (since it's marked as outdated and didn't show up in the code).